### ISSUE TITLE
How to even start?

### ISSUE THREAD

**Issue by Eddcapone** [owebia/magento2-module-advanced-shipping#81]
I am confused. The documentation only presents some code examples for adding shipping methods. But where do I even have to call the code? The documentation is missing a basic introduction.

**Comment by owebia** [comment]
Thanks for your feedback, you are right, we can improve that.

Best Regards,
A.L.

**Comment by Eddcapone** [comment]
Yes I figured out that you have to look in Store Configuration -> Sales -> Carrier Methods, but that was nowhere described.

**Comment by toledoroy** [comment]
> Carrier Methods



> Store Configuration -> Sales -> Carrier Methods

Geez, Thanks @Eddcapone ! 

Somehow they really did completely forget to mention that. 🤦‍♂️
